tests/openai/test_openai_autolog.py

```diff
@@ -445,25 +444,25 @@ async def test_autolog_with_registered_model_name(client):
 
 @pytest.mark.asyncio
 @pytest.mark.parametrize("log_models", [True, False])
-def test_autolog_use_active_run_id(client, log_models):
+async def test_autolog_use_active_run_id(client, log_models):
```

Comment:
Curious why this test only failed for 1.48.0.

Reply:
I have no idea 😅

Reply:
Is this test flaky?

Reply:
It's not flaky, it consistently fails on 1.48.0

Reply:
Interesting. There might be a bug in 1.48.0

Reply:
Yea I think so, but doesn't worth digging deeper 😅